script/import_scripts/socialcast/test/test_data.rb

```diff
@@ -7204,7 +7204,7 @@
                     "id": 18,
                     "url": "http://www.youtube.com/watch?v=Ce7ivrGMN0U",
                     "title": "World of Warcraft: The Molten Core",
-                    "description": "WORLD OF WARCRAFT COMING TO CONSOLES WITH MOLTEN CORE™ Blizzard Entertainment, Inc. announces its long-awaited return to console gaming with World of Warcraf...",
+                    "description": "WORLD OF WARCRAFT COMING TO CONSOLES WITH MOLTEN CORE™ Blizzard Entertainment, Inc. announces its long-awaited return to console gaming with World of Warcraft...",
```

Comment:
I should drop this change, the ellipsis is ok -- I generally skipped over html encoded forms of the same